dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::UsersController < ApplicationController
+  before_action :load_user
+
+  def load_user
+    user_id = params[:user_id]
+    @user = current_user
+    if user_id != 'me' && user_id != current_user.id
+      raise CanCan::AccessDenied
+    end
```

Comment:
nit: I'd prefer line 6(`@user = current_user`) at the end, since `@user` isn't needed when the error is raised, and also if we ever want to call this method in a different context it will return the user